src/server/vnparse.c

```diff
@@ -1489,10 +1489,6 @@ manage_resc_sum_values(enum resc_sum_action action, resource_def *resc_def, char
 			return (NULL);
 		}
 		resc_sum_values_size = 20;
-		for (k = 0; k < resc_sum_values_size; k++) {
-			rs = resc_sum_values;
-			rs[k].rs_def = NULL;
-		}
```

Comment:
We no longer want to initialize these values?

Reply:
Cause its already NULL due to the `calloc()` used in Line 1485